### PR TITLE
Templatefactory

### PR DESCRIPTION
**Pull request by doutriaux1**
@jccosta branch with merged conflicts fixed.

### CODE REVIEW

File: src/python/packages/amwg/amwg.py [hunk 1]
```diff
@@ -862,13 +862,9 @@ def customizeTemplates(self, templates, data=None, varIndex=None, graphicMethod=
...
-                    #line.color = ["red"]#, "blue", "salmon", "medium aquamarine", "orange", "chartreuse"]
-                    #line.type = ["dash"]#, "dot", "dash-dot", "long-dash", "solid", "dash"]
-                    #graphicMethod.line = line
-                                        
-                    graphicMethod.linecolor = 242
+                    graphicMethod.linetype = "dash"
+                    # The next repeated commands were necessary in Linux.
+                    graphicMethod.linecolor = "red"
                     graphicMethod.linewidth = 2
                     graphicMethod.linetype  = 1
```
Comment (jccosta) [inline src/python/packages/amwg/amwg.py]: @doutriaux1 you have the same command (linetype) executed two times for the same type: lines 865 and 869.

Comment (doutriaux1) [inline src/python/packages/amwg/amwg.py]: thanks I'll fix this!

**Comment by doutriaux1** [comment]
@jccosta with the latest changes in vcs it seems that your works looks a bit off see:
https://open.cdash.org/viewTest.php?onlyfailed&buildid=4507856
for example:
https://open.cdash.org/testDetails.php?test=474205321&build=4507856

To obtain the latest nightly vcs please run:

```
conda install -n nightly -c uvcdat/label/nightly -c uvcdat vcs cdms2
source activate nightly
```

**Comment by jccosta** [comment]
@doutriaux1,I will take a look on it. Something in vcs has changed. 
Thanks!

**Comment by jccosta** [comment]
@doutriaux1 
I ran a set of tests: https://open.cdash.org/viewTest.php?buildid=4508179
All plots seem to be ok, the problem is with the baseline images for some of the plots.
Can you confirm it?

**Comment by doutriaux1** [comment]
@jccosta you're right your test appear to be correct could you please just update the baselines? there's a script to do this: https://github.com/UV-CDAT/uvcmetrics/blob/master/src/python/devel/update_metrics_baselines.py

**Comment by jccosta** [comment]
@doutriaux1 
I will do it later today.

**Pull request merged**